zulipterminal/config/themes.py

```diff
@@ -52,154 +52,154 @@
 
 THEMES = {
     'default': [
-        (None,           'white',           ''),
-        ('selected',     'white',           'dark blue'),
-        ('msg_selected', 'white',           'dark blue'),
-        ('header',       'dark cyan',       'dark blue', 'bold'),
-        ('custom',       'white',           'dark blue', 'underline'),
-        ('content',      'white',           '',          'standout'),
-        ('name',         'yellow, bold',    ''),
-        ('unread',       'dark blue',       ''),
-        ('active',       'light green',     ''),
-        ('idle',         'yellow',          ''),
-        ('offline',      'white',           ''),
-        ('inactive',     'white',           ''),
-        ('title',        'white, bold',     ''),
-        ('time',         'light blue',      ''),
-        ('bar',          'white',           'dark gray'),
-        ('help',         'white',           'dark gray'),
-        ('emoji',        'light magenta',   ''),
-        ('reaction',     'light magenta, bold', ''),
-        ('span',         'light red, bold', ''),
-        ('link',         'light blue',      ''),
-        ('blockquote',   'brown',           ''),
-        ('code',         'black',           'white'),
-        ('bold',         'white, bold',     ''),
-        ('footer',       'white',           'dark red',  'bold'),
-        ('starred',      'light red, bold', ''),
-        ('category',     'light blue, bold', ''),
-        ('unread_count', 'yellow',          ''),
-        ('table_head',   'white, bold',      ''),
+        (None,             'white',           ''),
+        ('selected',       'white',           'dark blue'),
+        ('msg_selected',   'white',           'dark blue'),
+        ('header',         'dark cyan',       'dark blue', 'bold'),
+        ('custom',         'white',           'dark blue', 'underline'),
+        ('content',        'white',           '',          'standout'),
+        ('name',           'yellow, bold',    ''),
+        ('unread',         'light blue',      ''),
```

Comment:
Is this change from `dark blue` to `light blue` intentional? There are more similar changes in this commit.